ViewData/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using ManagerQueue;
+
+namespace ViewData
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Service start");
+            var queue = new QueueTransit("Config.json", "ConfigTransite.json");
+            while (true)
+            {
+                queue.TransitData();
+                queue.GetData();
+                if (queue.ServiceOperability() == false) break;
```

Comment:
it’s not necessary to use == operator for comparing boolean value. 
!queue.ServiceOperability() can be used.